environment.yml

```diff
@@ -1,11 +1,12 @@
-name: mibitracker-client
+name: mibilib
 channels:
 - conda-forge
 - defaults
 dependencies:
 - pip=19.1.1
 - python=3.7.3
 - pip:
+    - astroid==2.2.5
```

Comment:
uuuhhhh... what's this? And why does it conflict with the mibitof android version?

Reply:
You know how I feel about environment changes :-)